Barycentric_coordinates_2/examples/Barycentric_coordinates_2/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 
 project(Barycentric_coordinates_2_Examples)
 
-cmake_minimum_required(VERSION 3.1...3.15)
+cmake_minimum_required(VERSION 3.1...3.20)
```

Comment:
We could "upgrade" to `3.1...3.22`, because we test with CMake 3.22 in our testsuite.